scss/styles.scss

```diff
@@ -1,3 +1,14 @@
-.title {
-  color: red;
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+body {
```

Comment:
do not use empty rulesets

Reply:
And avoid as much as posible the ussage of tags in scss or css